apps/cowswap-frontend/src/legacy/components/Footer/index.tsx

```diff
@@ -3,8 +3,7 @@ import styled from 'styled-components/macro'
 
 import { Polling } from 'legacy/components/Header/Polling'
 import { useIsDarkMode } from 'legacy/state/user/hooks'
-
-import { footerImage } from 'common/constants/theme'
+import { footerImage } from 'legacy/theme/cowSwapAssets'
```

Comment:
This could stay

Reply:
what do you mean? this brings the normal CoWs back

Reply:
I see you refer to the fact is `legacy`, but that's where the file is 
![Uploading image.png…]()

Reply:
I see you refer to the fact is `legacy`, but that's where the file is


<img width="789" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/bbbfbbf4-33d4-4eb4-aa41-47c337624a78">

Reply:
I mean, the import was changed away from the `legacy` path.
I thought it was an improvement.